cute_sway_recorder/main.py

```diff
@@ -54,6 +55,7 @@ def wf_recorder(
     if isinstance(selection, SelectedScreen):
         params.append("--output")
         params.append(selection)
+    params.append(flags.strip())
```

Comment:
`strip()` is needed to be safe.

if whitespace is present the flags won't work:

`wf-recorder: unrecognized option '--force-yuv`